resources/less/vars.less

```diff
@@ -28,6 +28,7 @@
 @input-font: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
 @modal-shadow: 0 1px 3px 0 rgba(0,0,0,0.30);
 @separator-color: #E6E6E6;
-
 @sg-color-red: #e44e4e;
 @sg-color-gray: #aeaeae;
+@disabled-gray: #9b9b9b;
```

Comment:
Are the style guide colors? If we do add new colors I think we should just use those. I wouldn't use semantic names for colour variables either; they're general colours and the semantics should be clear at the place of use.